app/views/layouts/_footer.html.erb

```diff
@@ -29,7 +29,7 @@
         <li class="nav-item" data-turbolinks="false">
           <%= link_to 'Ping Thing', ping_things_path(network: params[:network]), class: 'nav-link' %>
         </li>
-        <li class="nav-item" data-turbolinks="false">
+        <li class="nav-item" data-turbolinks="true">
```

Comment:
you can delete data-turbolinks as "true" is the default behavior

Reply:
Yeah, you have totally right.